### DISCUSSION TITLE
Option of adding AssertFail to FB_TestSuite

### DISCUSSION THREAD

**Discussion by ZigaJavornik** [tcunit/TcUnit#245] [Ideas]
Is there an option to possibly add an AssertFail method with only message input that automatically fails the test when called? I think it might be a neat addition.

Best regards

**Comment by sagatowski** [comment]
What's the usecase for this?

**Reply by ZigaJavornik** [reply]
One usecase would be leaving unfinished tests, and just putting an AssertFail to never forget about finishing them. There are some other uses, possibly in 4026 where exceptions and Try Catch are introduced, if I am not mistaken.

**Reply by sagatowski** [reply]
You can achieve what you want by either:
1. `AssertTrue(False)` or
2. `TEST('DISABLED_MyTest')`, see [here](https://tcunit.org/#/faq?id=_2-how-can-i-disableignore-a-test)
